hail/python/hailtop/batch/batch.py

```diff
@@ -90,7 +90,8 @@ def __init__(self,
                  default_memory: Optional[str] = None,
                  default_cpu: Optional[str] = None,
                  default_storage: Optional[str] = None,
-                 default_timeout: Optional[Union[float, int]] = None):
+                 default_timeout: Optional[Union[float, int]] = None,
+                 default_shell: str = None):
```

Comment:
default_shell: str = '/bin/bash'?

Reply:
Either seems fine, but None has the advantage you can tell if the user specified the value or not.  It doesn't seem relevant here, but it can be.